supabase/functions/oauth/encrypt-config.ts

```diff
@@ -87,14 +89,17 @@ export async function encryptConfig(req: Record<string, any>) {
       responseData.includes(oauth2_client_id)) ||
     (typeof oauth2_client_secret === "string" &&
       oauth2_client_secret.length > 0 &&
-      responseData.includes(oauth2_client_secret))
+      responseData.includes(oauth2_client_secret)) ||
+    (typeof oauth2_injected_values === "object" &&
+      oauth2_injected_values !== null &&
+      _.some(_.values(oauth2_injected_values), (value: string) => responseData.includes(value)))
```

Comment:
The challenge with this is that if someone is injecting something short, we may take issue with it. But I suppose any secret that needs to be secure must be long-enough™️, so perhaps I'm overthinking it.

Reply:
My understanding is that the only use case we have for `oauth2_injected_values` at the moment is the developer token that's required by Google. Given that, I think this is fine. If we have need to inject non-sensitive data later, we can figure out how to address that then.